tests/services/test_spark.py

```diff
@@ -1,14 +1,20 @@
 # go three folders up
 import logging
-from ...app.services.spark_oceandb import *
 
+from ...app.services.spark_oceandb import FileLoader, SparkSession
 
 
-logging.info("Starting Spark Session!")
-try:
-    spark = FileLoader(SparkSession())
-    # load excel:Returns dataframe and its schema
-    _,schema = spark.load_excel("./data/SaleData.xlsx")
-    logging.info('Code Executed Successfully')
-except Exception as e:
-    logging.warning(e)
\ No newline at end of file
+class TestLoadData:
+    def load_excel(file_path: str = './data/SaleData.xlsx'):

```

Comment:
The name of the function should start from `test_*`. This will tell test runner to execute this function as a test.

Reply:
done